pythonsd/templates/pythonsd/base.html

```diff
@@ -23,9 +23,14 @@
 
   <body>
     <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-      <a class="navbar-brand" href="{% url 'index' %}">
-        <img src="{% static 'img/sandiegopython-logo.svg' %}" alt="San Diego Python Home">
+    {% url 'index' as index_url %}
+    {% if request.path == index_url %}
+		<img src="{% static 'img/sandiegopython-logo.svg' %}" id="navbar-logo" class="navbar-brand" alt="San Diego Python Home">
+    {% else %}
+      <a class="navbar-brand" href="{{ index_url }}">
+		<img src="{% static 'img/sandiegopython-logo.svg' %}" id="navbar-logo" alt="San Diego Python Home">
       </a>
+    {% endif %}
```

Comment:
This is the alternative using `href="#"`. If this is accepted, please also revert the changes to `_theme.scss`:

```suggestion
      {% url 'index' as index_url %}
      <a class="navbar-brand" href="{% if request.path != index_url %}{{ index_url }}{% else %}#{% endif %}">
		    <img src="{% static 'img/sandiegopython-logo.svg' %}" id="navbar-logo" alt="San Diego Python Home">
      </a>
```